src/Components/Aspire.Qdrant.Client/AspireQdrantExtensions.cs

```diff
@@ -80,6 +82,41 @@ private static void AddQdrant(
             builder.Services.AddKeyedSingleton(serviceKey, (sp, key) => ConfigureQdrant(sp));
         }
 
+        if (!settings.DisableHealthChecks)
+        {
+            // use http client for register health check
+            var httpClientName = serviceKey is null ? "qdrant-healthchecks" : $"qdrant-healthchecks_{connectionName}";
+            builder.Services.AddHttpClient(
+                 httpClientName,
+                 client =>
+                 {
+                     if (settings.Endpoint is null)
+                     {
+                         throw new InvalidOperationException(
+                                 $"A QdrantClient could not be configured. Ensure valid connection information was provided in 'ConnectionStrings:{connectionName}' or either " +
+                                 $"{nameof(settings.Endpoint)} must be provided " +
+                                 $"in the '{DefaultConfigSectionName}' or '{DefaultConfigSectionName}:{connectionName}' configuration section.");
+                     }
+
+                     client.BaseAddress = settings.Endpoint;
+
+                     if (settings.Key is not null)
+                     {
+                         client.DefaultRequestHeaders.Add("Api-Key", settings.Key);
+                     }
```

Comment:
According to https://qdrant.tech/documentation/guides/monitoring/?q=health, you shouldn't need to specify the Api-Key.

> Regardless of whether an [API key](https://qdrant.tech/documentation/guides/security/#authentication) is configured, the endpoints are always accessible.